### PR TITLE
Added to contribution docs with Test Automation steps

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Updated contributions with test automation test steps.

### CODE REVIEW

File: contributions/docs/TestAutomation.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+How can I contribute to Cypress Test Suite.
```
Comment (mohanarpit) [inline contributions/docs/TestAutomation.md:1]: Please format this document in markdown. Right now the entire text is rendered as a single line.

File: contributions/docs/TestAutomation.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+1.Follow the initial document to setup Appsmith locally.
```
Comment (mohanarpit) [inline contributions/docs/TestAutomation.md:3]: Is it possible to link to the setup document here? Make it easy for people to link-click.

File: contributions/docs/TestAutomation.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+2.Once we are have the setup in place, all dependencies will be available in local for test execution.
+3.Change directory to appsmith/app/client .
```
Comment (mohanarpit) [inline contributions/docs/TestAutomation.md:5]:
```suggestion
3.Change directory to appsmith/app/client .
```
```suggestion
3. Change your directory to `appsmith/app/client`.
```

Using backticks ensures paths rendered properly and easily in a document

File: contributions/docs/TestAutomation.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+4.Cypress tests are located within, cypress folder.
+5.All the test spec must be within the integration folder only I.e appsmith/app/client/cypress/integration
+6.You can create folders within cypress folder but make sure you place the spec within integrations folder.
+7.Cypress tests can be executed in 2 modes run mode and open mode.
+If you are willing to execute tests in headless mode through command line follow run mode 
+[Refer here https://docs.cypress.io/guides/guides/command-line.html#How-to-run-commands]
```
Comment (mohanarpit) [inline contributions/docs/TestAutomation.md:11]: Please hyperlink. Check markdown syntax for creating hyperlink urls.

**Comment by mohanarpit** [comment]
@NandanAnantharamu Thanks for documenting this process. My biggest feedback here is to improve the formatting & grammar. Please use a service such as Grammarly which will automatically improve the grammar. Using a Word doc will improve the formatting. In addition to that, please use a markdown renderer (also available in VSCode) to see how the page will render. These little things will go a long way in ensuring that the document is read & used.

File: contributions/docs/TestAutomation.md (hunk 1 above)
Comment (NandanAnantharamu) [reply contributions/docs/TestAutomation.md]: done

File: contributions/docs/TestAutomation.md (hunk 3 above)
Comment (NandanAnantharamu) [reply contributions/docs/TestAutomation.md]: done

File: contributions/docs/TestAutomation.md (hunk 2 above)
Comment (NandanAnantharamu) [reply contributions/docs/TestAutomation.md]: done

**Review by mohanarpit — APPROVED** [review]

**Pull request merged**